examples/AccountManagement/GetAccountHierarchy.cs

```diff
@@ -0,0 +1,218 @@
+// limitations under the License.
+
+using System.Collections;
+using System.Collections.Generic;
+using Google.Ads.GoogleAds.V3.Resources;
+using Google.Api.Gax;
+
+namespace Google.Ads.GoogleAds.Examples.V3
+{
+    using Lib;
+    using Google.Ads.GoogleAds.V3.Errors;
+    using Google.Ads.GoogleAds.V3.Services;
+    using System;
+
+    /// <summary>
+    /// This example gets the account hierarchy of the specified manager account. If you don't specify
+    /// manager customer ID, the example will instead print the hierarchies of all accessible customer
+    /// accounts for your authenticated Google account.
+    /// Note that if the list of accessible customers for your authenticated Google account includes
+    /// accounts within the same hierarchy, this example will retrieve and print the overlapping
+    /// portions of the hierarchy for each accessible customer.
+    /// </summary>
+    public class GetAccountHierarchy : ExampleBase
+    {
+        // private const string MANAGER_CUSTOMER_ID = "3955750366";//"INSERT_MANAGER_CUSTOMER_ID_HERE";
+        private const int PAGE_SIZE = 1000;
+
+        /// <summary>
+        /// Main method, to run this code example as a standalone application.
+        /// </summary>
+        /// <param name="args">The command line arguments.</param>
+        public static void Main(string[] args)
+        {
+            GetAccountHierarchy codeExample = new GetAccountHierarchy();
+            Console.WriteLine(codeExample.Description);
+            long customerId = long.Parse("INSERT_CUSTOMER_ID_HERE");
+            
+            try
+            {
+                codeExample.Run(new GoogleAdsClient(), customerId);
+            }
+            catch (GoogleAdsException adsException)
+            {
+                Console.WriteLine("Failure:");
+                Console.WriteLine($"Message: {adsException.Message}");
+                Console.WriteLine($"Failure: {adsException.Failure}");
+                Console.WriteLine($"Request ID: {adsException.RequestId}");
+                throw;
+            }
+        }
+
+        /// <summary>
+        /// Returns a description about the code example.
+        /// </summary>
+        public override string Description =>
+            "This code example gets the account hierarchy of a specified manager account. " +
+            "If you don't specify a manager customer ID, the example will instead print the hierarchies " +
+            "of all accessible customer accounts for your authenticated Google account.";
+
+        /// <summary>
+        /// Runs the code example.
+        /// </summary>
+        /// <param name="googleAdsClient">The Google Ads client instance.</param>
+        /// <param name="customerId">The customer ID.</param>
+        public void Run(GoogleAdsClient googleAdsClient, long? customerId = null)
+        {
+             // googleAdsServiceClient = googleAdsClient.GetService(Services.V3.GoogleAdsService);
```

Comment:
I don't think this is supposed to be commented out.

Reply:
Fixed.